mesmerize_napari/napari1d_manager.py

```diff
@@ -111,8 +106,17 @@ def callback(layer, event):
 
     def update_visible_components(self):
         edge_colors, face_colors = self.get_colors()
-        self.spatial_layer.edge_color = edge_colors
-        self.temporal_layer.color = edge_colors
+        og_num, current_num = np.shape(edge_colors)[0], np.shape(self.spatial_layer.edge_color)[0]
+        ixs_comps = self.cnmf_obj.estimates.idx_components
+        if og_num != current_num:
```

Comment:
some comments on what this does, removes all the spatial and plots according to new data? How was it before, I thought at some point we were playing with alphas or am I remembering wrong?

Reply:
Could do with opencv to change just colors. Decided not to, napari should fix shapes layer